### ISSUE TITLE
The pypi distribution of this package is severely out of date

### ISSUE THREAD

**Issue by idreyn** [allelos/vectors#11]
Although it seems like this useful package is still under development, there hasn't been a published release since December 2016. This version has some basic errors, for instance:
```
v1 = Vector(1, 0, 0)
v2 = Vector(0, 1, 0)
v1.sum(v2) // returns Vector(1.0, 1.0, 1.0)
```
It would be nice to get an updated version of this onto pypi — let me know if there's any way I can help to do so.

**Comment by allelos** [comment]
I agree with you, but I have been very busy and I haven't got any time to work on this package.

You are welcomed to contribute and it is greatly appreciated

**Comment by austinphilp** [comment]
@allelos I'd be happy to help out and get the PyPI distribution up to date

**Comment by austinphilp** [comment]
@allelos do you have an email I can contact you with so that we can coordinate on releasing a new version to PyPI?

**Comment by allelos** [comment]
@austinphilp p.paliagkas@gmail.com

**Comment by HappyDustbunny** [comment]
Addition is still not commutative...
v1.sum(v2) =/= v2.sum(v1)

**Comment by allelos** [comment]
It's time ⌚

**Comment by JarrettR** [comment]
How can I help with this? Submit a PR with an updated version number?

**Comment by austinphilp** [comment]
@JarrettR  I believe that @allelos just needs to submit to increment the version number, create a new tag, and then submit the new version on pypi.

@allelos if you still don't have time to do any work on this package, perhaps consider transferring ownership of the pypi repo to another contributor so that they can make a fork and publish that fork on Pypi

**Comment by JarrettR** [comment]
Mostly asking because this is an excellent library with few dependencies that is simple and does exactly what it says on the tin. I like it, so I want to reduce friction for installation as much as possible :)

**Comment by threeal** [comment]
Hi @allelos, I think you should either update the pypi distribution or transfer ownership as this package still has a problem in Python 3 (https://github.com/allelos/vectors/issues/8)
